Task 4/google-search-engine-clone.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Google</title>
+    <link
+      rel="icon"
+      href="https://th.bing.com/th/id/OIP.lp6AjI9vfG4iLCicENccwQHaHa?w=150&h=150&c=7&r=0&o=5&dpr=1.25&pid=1.7"
+    />
+    <link rel="stylesheet" href="/google-search-engine-clone.css" />
+    <script
+      src="https://kit.fontawesome.com/992348ea18.js"
+      crossorigin="anonymous"
+    ></script>
```

Comment:
Explain this

Reply:
emported mic menu bar and and search icon from this website.